controllers/cluster_controller.go

```diff
@@ -121,6 +121,11 @@ func (r *FoundationDBClusterReconciler) Reconcile(ctx context.Context, request c
 	}
 	defer adminClient.Close()
 
+	err = adminClient.CleanupOldLogs()
```

Comment:
I think we should be adding a context.WithTimeout() for this with a configurable timeout, just in case the underlying fs/disk is slow for any reason. That way we can at least ensure we won't block the loop for more than `timeout`.

Reply:
In the future I also think we should have a flag to control whether we keep or discard older files for the rare case that someone needs to debug using older fdbcli trace files.

Reply:
Added a flag to enable/disable this feature (default enabled).